### DISCUSSION TITLE
YouTube's rss has changed

### DISCUSSION THREAD

**Discussion by vejobo9414larland** [TeamNewPipe/NewPipe#10898] [Questions]
Hello.
The mechanism for receiving data from YouTube pages via RSS stopped working.
please see what can be done.

**Comment by vejobo9414larland** [comment]
Already working)

**Reply by opusforlife2** [reply]
So it was a temporary problem?

**Reply by vejobo9414larland** [reply]
Yes. 3 hours after discovery it was already working as before. Looks like someone on Google was playing with the settings)
